package.json

```diff
@@ -27,7 +27,8 @@
         "@patternfly/react-core": "^4.75.2",
         "@patternfly/react-table": "^4.19.5",
         "fuse.js": "^6.4.3",
-        "get-value": "^3.0.1"
+        "get-value": "^3.0.1",
+        "react-tag-autocomplete": "^5.12.0"
```

Comment:
Latest version of react-tag-autocomplete is 6.1.0, is it possible to use the latest?  It makes it easier for security updates.

https://www.npmjs.com/package/react-tag-autocomplete

Reply:
Not sure if this is my lack of knowledge of typescript but i require both `react-tag-autocomplete` and `@types/react-tag-autocomplete`. unfortunately the `@types/react-tag-autocomplete` dep is only at version `5.12.0` and if I use the most up-to-date version `6.1.0` there are syntax errors around the type definitions.

Reply:
This will cause an issue as the ui-components library is on the latest versions of everything and is setup to try to auto update packages. We need to see if we can find a solution.